README.md

```diff
@@ -1,59 +1,54 @@
 # Karmada-dashboard
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/kubernetes/dashboard/blob/master/LICENSE)
 
-Karmada Dashboard is a general-purpose, web-based control panel for Karmada which is a multi-cluster management project.
-![image](docs/images/dashboard.png)
+English version|[中文版](README_cn.md)
 
-## Latest release
-🎉 This is the first release for Karmada-dashboard!
+Karmada Dashboard is a general-purpose, web-based control panel for Karmada which is a multi-cluster management project.
+![image](docs/images/readme-dashboard-en.png)
 
 ## 🚀QuickStart
 
 ### Prerequisites
+- Multi kubernetes clusters[optional]: in case of testing, you can start kubernetes clusters with kind. If you already had kubernetes clusters in production, you can skip this step.
+- Control-plane of Karmada[optional]:  in case of testing, you can follow this [tutorial](https://github.com/karmada-io/karmada/blob/master/README.md#install-karmada-control-plane) to set up a minimal environment with Karmada. If you are already installed Karmada, also this step can be skipped.
```

Comment:
```suggestion
You need to have a Karmada installed on Kubernetes(aka. `host cluster`) and the [karmadactl](https://karmada.io/docs/installation/install-cli-tools#install-karmadactl) or
kubectl command-line tool must be configured to communicate with your host cluster and Karmada control plane.

If you don't already have the Karmada, you can launch one by following this [tutorial](https://karmada.io/docs/installation/#install-karmada-for-development-environment).
```
How about this?

Reply:
It's ok, the cn version of readme also needs to be changed

Reply:
So, let's do it.

Reply:
I've add the translation to cn version, the suggestion can be commited.